src/chart/generator/plot.cpp

```diff
@@ -129,6 +129,11 @@ Buckets Plot::generateMarkers()
 	for (auto &&[ix, mid] : options->markersInfo)
 		map.emplace(mid, ix);
 
+	bool rectangleSpacing{true};
+	if (auto &&sp = style.plot.marker.rectangleSpacing;
+	    sp && sp->get().has_value())
+		rectangleSpacing = *sp->get() > 0.0;
+
```

Comment:
Why is this needed? Cannot we handle this only at rendering? Marker::spacing supposed to be a style independent factor

Reply:
After checking the test cases, we concluded that this is not needed.